### PR TITLE
Add ocproxy formula

### PR DESCRIPTION
**Pull request by r4um**

### CODE REVIEW

File: Library/Formula/ocproxy.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class Ocproxy < Formula
+  desc "user-level SOCKS and port forwarding proxy"
+  homepage "https://github.com/cernekee/ocproxy"
+  url "https://github.com/cernekee/ocproxy/archive/v1.50.tar.gz"
+  sha256 "b061d59e0b5729d7a8950d8d4e0004a9739c092a4d5daf58a3bc61b177b24e4f"
+
+  head do
+    url "https://github.com/cernekee/ocproxy.git"
+  end
```
Comment (mikemcquaid) [inline Library/Formula/ocproxy.rb]: Can just do `head "https://github.com/cernekee/ocproxy.git"` when it's just a URL.

File: Library/Formula/ocproxy.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    system "./autogen.sh"
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    `#{bin}/ocproxy 2>&1` =~ /VPNFD.is.not.set/
```
Comment (mikemcquaid) [inline Library/Formula/ocproxy.rb]: Use `assert_match` here instead. Also, as it's a proxy I appreciate it's probably not but: can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Comment (r4um) [inline Library/Formula/ocproxy.rb]: > can this test be modified to do something more substantial than e.g. --version or --help?

Yeah wanted to do the same, sadly no flags like that exist :(.

Comment (mikemcquaid) [inline Library/Formula/ocproxy.rb]: No worries.

**Comment by apjanke** [comment]
Looks good now. Would you mind squashing this to a single commit, so we end up with a tidy commit history and no non-working commits?

**Comment by r4um** [comment]
@apjanke done.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/fb2900bacb4682b4e2521d16a27764ab82bfbabf. Thank you for your contribution to Homebrew @r4um; we appreciate it! :smiley_cat:

**Pull request closed without merging**